api/napi/index.ts

```diff
@@ -152,6 +152,8 @@ export interface ComponentHandle {
     show();
     hide();
     get window(): Window;
+    send_mouse_click(x: number, y: number);
+    send_keyboard_string_sequence(s: String);
```

Comment:
Is there a way to only have that for the tests?  I actually don't think it should be part of the public interface

Reply:
done